### PR TITLE
Filtering posts

### PR DESCRIPTION
**Pull request by danilkolikov**
- Now it's possible to select networks from sidebar

### CODE REVIEW

File: app/src/main/java/ly/loud/loudly/ui/TitledFragment.java [hunk 1]
```diff
@@ -8,17 +8,38 @@
...
         ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
         if (actionBar != null) {
-            actionBar.setTitle(getTitle());
+            actionBar.setTitle(title);
         }
     }
+
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
```
Comment (ZeRoGerc) [inline app/src/main/java/ly/loud/loudly/ui/TitledFragment.java]: Nullability. And Icepick first I think)

File: app/src/main/java/ly/loud/loudly/ui/feed/FeedFragment.java [hunk 2]
```diff
@@ -129,6 +132,20 @@ public void onDestroyView() {
         super.onDestroyView();
     }
 
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
```
Comment (ZeRoGerc) [inline app/src/main/java/ly/loud/loudly/ui/feed/FeedFragment.java]: Nullability

**Comment by ZeRoGerc** [comment]
ok
but merge

**Comment by danilkolikov** [comment]
fixed

**Pull request merged**